README.md

```diff
@@ -1,6 +1,73 @@
 # Mina zkApp: Mina Mastermind
 
-This template uses TypeScript.
+## Understanding The Game
+
+The game has two players, a code master and code breaker. 
+
+The code master generates 4 secret digits in a set sequence. Digits can be between 1-9 and the digits must all be different. 
+Then, each turn, the code breaker tries to guess the code master's digits, who then gives the number of matches.  
+If the matching digits are in their right positions, they are "hits", if they in different positions, they are "blows". 
+
+Example:
+
+Code master private solution: 4 2 7 1 <br />
+Code breaker's public solution: 1 2 3 4 <br />
+Answer: 1 hit and 2 blows. (The hit is "2" in the second position, the blows are "4" and "1".) <br />
```

Comment:
Actually 2 is a hit and 1 is blow.

Reply:
Its saying that there is 1 hit and 2 blows in total... With 2 being a hit whilst 4 and 1 being blows. 

I think thats is correct right?

Reply:
I see, there's some lexical confusion!
- The digit `2` in the example is in the same position and equal and hence it is a hit
- The digit `1` in the example exists in the solution but it's not in the correct position hence it is a blow

Reply:
(The hit is "2" in the second position, the blows are "4" and "1")

Reply:
correct, I overlooked the digit `4` :+1: